src/components/Navbar.css

```diff
@@ -1,31 +1,50 @@
-header {
+header,
+nav,
+ul {
   display: flex;
-  justify-content: space-between;
-  padding: 10px 10px 20px 10px;
 }
 
-nav {
+header {
   display: flex;
-  justify-content: space-between;
+  padding: 10px 10px 20px 10px;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fff;
+  align-items: center;
+  padding-inline-start: 40px;
+  border-bottom: 1px solid #e8e8e8;
 }
 
 h1 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
   color: #0290ff;
 }
 
-ul {
+.nav-links {
   display: flex;
-  align-items: center;
-  list-style: none;
-  gap: 10px;
+  flex-grow: 1;
+  list-style-type: none;
+  grid-gap: 40px;
+  gap: 40px;
+  letter-spacing: 1.9px;
 }
```

Comment:
Your design looks good. Kindly let the currently navigated link be highlighted as shown on the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/215008535-b4dc8b0e-fdef-4187-ab08-3a496fd19ed4.png)

Design guide:
![image](https://user-images.githubusercontent.com/20554583/215008551-dc2718c9-303b-4c28-9d32-fb7d7d3d35cd.png)